domain/schema/controller.go

```diff
@@ -445,3 +448,29 @@ CREATE TABLE model_defaults (
 );
 `)
 }
+
+func autocertCacheSchema() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE autocert_cache (
+    name 		TEXT PRIMARY KEY,
```

Comment:
Can we change this to a uuid and then make the name as a name a unique constraint?

Reply:
sure, done